### DISCUSSION TITLE
Optuna restrict to only to the first trial and display it as the best one

### DISCUSSION THREAD

**Discussion by Arch-suzuki-MB** [optuna/optuna#5461] [Q&A] [answered]
### Don't use GitHub Issues to ask support questions.

hi everyone, I'm trying to fine tune the parameter of A2C from stable baseline with optuna to solve highway-fast-v0 env from gym but the problem is optuna achieve only one trial and it show me as the best trial. Actually I'm stuck I don't know who's responsible of this because I try the same code but in cartPole env and it was working normally I got many trials and then the hyperparameters of the best. 

this is my code : https://colab.research.google.com/drive/1ohJ4-guWyjvgpYvQ_DAuDSTNKEz3-1gL?usp=sharing 

can any one @araffin , @nzw0301 , @hvy or any one from optuna team to help and thanks a lot

**Comment by nzw0301** [comment]
As written in the issue template, please use GitHub discussions to ask a question, https://github.com/optuna/optuna/discussions.

**Comment by nzw0301** [comment, marked as answer]
Since you use the  timeout argument, if the first trial takes more than timeout seconds, the reported result sounds normal. Not set timeout or set much larger value would resolve it.
